xml/System.Runtime/DependentHandle.xml

```diff
@@ -106,9 +152,17 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets a value indicating whether this instance was constructed with <see cref="M:System.Runtime.DependentHandle.#ctor(System.Object,System.Object)" /> and has not yet been disposed.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if this instance was constructed with <see cref="M:System.Runtime.DependentHandle.#ctor(System.Object,System.Object)" /> and has not yet been disposed; <see langword="false" /> otherwise.</value>
```